pages/index.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+function useNW () {
+    if (typeof nw !== "undefined") {
+        return nw;
+    } else {
+        return null;
+    }
+}
 
 function App() {
+
+    const nw = useNW();
+
     return (
         <div>
             NW.js Next.js Example
             <br />
             <ul>
-                <li>NW.js: {nw.require('process').version['nw']}</li>
-                <li>Chromium: {nw.require('process').version['chromium']}</li>
-                <li>Node.js: {nw.require('process').version['node']}</li>
-                <li>V8: {nw.require('process').version['v8']}</li>
+                <li>NW.js: {nw && nw.require('node:process').versions['nw']}</li>
```

Comment:
During development, this works fine. But after building web app, the code after `nw &&` is stripped out by the compiler. Ideally, we don't want this to happen.